shark/examples/shark_inference/mega_test.py

```diff
@@ -52,14 +52,22 @@ def forward(self, input):
     base_model_id=None,
     model_name="mega_shark",
     precision=None,
-    return_mlir=False,
+    return_mlir=True,
```

Comment:
Because I'm using the compile API which is customised for StableDIffusion - it leads to a key error.
This change would avert the same.

Reply:
but we should fix the api to not be customized

Reply:
Okay, sure - this was just a quick workaround.
I'll work on it and create another API instead which we can use with any other model seamlessly. Will undo this change thereafter.